src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -141,6 +141,10 @@ private function getControllerError($callable)
             } else {
                 return sprintf('Function "%s" does not exist.', $callable);
             }
+
+            if (0 === strpos($callable, '\\')) {
```

Comment:
Why not trim all the time? Will this introduce a big performance overhead?

Reply:
I would instead always trim.